pkgs/applications/virtualization/open-vm-tools/default.nix

```diff
@@ -21,9 +22,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook makeWrapper pkgconfig ];
   buildInputs = [ fuse glib icu libdnet libmspack openssl pam procps xercesc ]
-      ++ lib.optionals withX [ gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
+      ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
-  patches = [ ./recognize_nixos.patch ];
+  patches = [ ./recognize_nixos.patch ./find_gdk_pixbuf_xlib.patch ];
```

Comment:
can reference your upstream patch https://github.com/vmware/open-vm-tools/pull/438 in a comment?

Reply:
Done